sycl/plugins/cuda/pi_cuda.cpp

```diff
@@ -2215,7 +2275,7 @@ pi_result cuda_piQueueFinish(pi_queue command_queue) {
 /// \return PI_SUCCESS
 pi_result cuda_piextQueueGetNativeHandle(pi_queue queue,
                                          pi_native_handle *nativeHandle) {
-  *nativeHandle = reinterpret_cast<pi_native_handle>(queue->get());
+  *nativeHandle = reinterpret_cast<pi_native_handle>(queue->get()); // TODO joe: two streams...
```

Comment:
This is open to discussion in the CUDA backend document, for now just return the normal stream...